### PR TITLE
Set up linter

### PR DESCRIPTION
**Pull request by bjohansebas**
I've tried using ts-standard as a linter, but it has caused quite a few issues. We can switch to ts-standard later before moving the repository to Express, but for now, I don't want to keep spending time on this. There are more important things to do here than setting up a linter

### CODE REVIEW

File: biome.json [hunk 1]
```diff
@@ -0,0 +1,38 @@
+{
```
Comment (kjugi) [inline biome.json:1]: do we want to add `.vscode` directory with recommended plugins and leave a link to this doc: https://biomejs.dev/reference/vscode/ in README so people can follow the configuration steps from here?

If you think it's super temporary to have this here - I'm fine with resolving it for now

File: transforms/__testfixtures__/magic-redirect/location.output.ts [hunk 2]
```diff
@@ -2,6 +2,6 @@ import express from 'express'
 
 const app = express()
 
-app.get('/', function (req, res) {
+app.get('/', (req, res) => {
```
Comment (kjugi) [inline transforms/__testfixtures__/magic-redirect/location.output.ts:5]: let's exclude `__testfixtures__` files. I'm already using both syntaxes [here](https://github.com/bjohansebas/codemod/blob/main/transforms/__testfixtures__/magic-redirect/location.output.ts#L9-L12)

Comment (bjohansebas) [reply transforms/__testfixtures__/magic-redirect/location.output.ts:5]: good catch

File: biome.json (hunk 1 above)
Comment (bjohansebas) [reply biome.json:1]: With .vscode it will be enough, but I would prefer not to add this folder for the moment.

**Pull request merged**